docs/trace/extending-the-sdk/README.md

```diff
@@ -119,26 +119,25 @@ Contrib](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/mai
 If you are writing an instrumentation library yourself, use the following
 guidelines.
 
-### Writing own instrumentation library
+### Writing a custom instrumentation library
 
-This section describes the steps required to write your own instrumentation
+This section describes the steps required to write a custom instrumentation
 library.
 
-*If you are writing a new library or modifying an existing library, the
-recommendation is to use [ActivitySource API/OpenTelemetry
+**Note** If you are writing a new library or modifying an existing library the
```

Comment:
In general, we follow the word "Note" by a colon.

Reply:
I'll do as a follow-up so the diff doesn't get all messy. In metric docs looks like some mixed use of `Note:` vs `NOTE:` is there a preference for casing?

Reply:
I don't think we have established any preference for casing. You could probably standardize that in the follow-up PR. I'm okay with both `Note:` and `NOTE:`.